plugins/module_utils/hmc_resource.py

```diff
@@ -76,8 +99,16 @@ def checkHmcUpandRunning(self, rebootStarted=False, timeoutInMin=12):
         waited = 0
         pingSuccess = False
         while waited < WAIT_UNTIL_IN_SEC:
-            ping_state = self.pingTest(self.hmcconn.ip)
+            # Check the hmc status using 'SSH' every 5th interval only in password less authentication
```

Comment:
Elaborate this by explaining the situation of ICMP disable customer environment, this routine helps to confirm the active remote HMC by SSHing to it